### DISCUSSION TITLE
Welcome to NDBC Discussions!

### DISCUSSION THREAD

**Discussion by supermanzer** [supermanzer/NDBC#22] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by piratepheasant** [comment]
I'm Sunni, and I use NDBC data for model validation.

I'm writing because I want to verify the time zone of the data I'm pulling.

I read on the NDBC Measurement Descriptions and Units page: https://www.ndbc.noaa.gov/measdes.shtml that both Realtime and Historical files show times in UTC only.

I took a look at the NDBC.py code, and it looks like the data I'm pulling (stdmet) is coming from files that are accessed.

If that's the case, I would imagine the remark about Realtime and Historical files (UTC) applies to the data I'm pulling.

I just want to confirm for my own use and for the benefit of others.

Thanks!

**Reply by supermanzer** [reply]
Hi :wave: 

This package makes no modification of the date/time values in the `.txt` files loaded from the NDBC web server beyond converting them into Python datetime objects.  So the values present are mirroring the text values provided by NDBC.
